### DISCUSSION TITLE
2024-09-10 오프라인 회의

### DISCUSSION THREAD

**Discussion by nyj001012** [jiphyeonjeon-42/backend-nest#39] [General]
.참여자 : yena, minsepar

지난 시간 목표설정 중간점검
이슈 공유

**Comment by parkj12b** [comment]
nest_practice 리포: https://github.com/parkj12b/nest_practice.git
## 진행도
- v1 user nest 로 옮기는거 진행하기 - 한개도 못함
### 목표
- v1 user 계속 진행하기

**Comment by nyj001012** [comment]
## 진행도
### TypeORM 연동
- conflict resolve 후 오늘 내 merge 예정 #38 

## 목표
- histories 스웨거 작성

**Comment by scarf005** [comment]
## 진행도

swagger 설정 및 CI/CD 설정

- #32 
- #35
- #36 
- #37 

## 목표

- 명세 작성?
- 컴포즈 파일 추가?
